### PR TITLE
Remove unnecessary environment check in lib/app.js

### PR DESCRIPTION
**Pull request by teleginzhenya**
Hello :)

I have removed unnecessary environment check from [`lib/app.js`](https://github.com/zeit/next.js/blob/canary/lib/app.js#L81-L85) (Lines 81-85), since [`lib/utils.js`](https://github.com/zeit/next.js/blob/canary/lib/utils.js#L2-L4) (Lines 2-4) already have it.

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -79,9 +79,7 @@ export class Container extends Component {
 }
 
 const warnUrl = execOnce(() => {
-  if (process.env.NODE_ENV !== 'production') {
-    warn(`Warning: the 'url' property is deprecated. https://err.sh/next.js/url-deprecated`)
-  }
+  warn(`Warning: the 'url' property is deprecated. https://err.sh/next.js/url-deprecated`)
```
Comment (timneutkens) [inline lib/app.js]: This is totally necessary. We only want to log this in development.

Comment (teleginzhenya) [reply lib/app.js]: [`lib/utils.js`](https://github.com/zeit/next.js/blob/canary/lib/utils.js#L2-L4) (Lines 2-4) handles it, doesn’t it?

Comment (timneutkens) [reply lib/app.js]: Also this makes sure we compile it out using webpack, instead of leaving the warn code in production it's compiled away.

Comment (teleginzhenya) [reply lib/app.js]: Was thinking that it handles and can remove unnecessary check. Sorry :)

**Pull request closed without merging**